sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py

```diff
@@ -6,58 +6,60 @@
 # --------------------------------------------------------------------------
 import pytest
 from datetime import timedelta
-from azure.communication.identity import CommunicationIdentityClient
 from azure.communication.identity import CommunicationTokenScope
-from devtools_testutils import AzureRecordedTestCase, is_live, recorded_by_proxy
-from _shared.utils import get_http_logging_policy
-from devtools_testutils.fake_credentials import FakeTokenCredential
+from devtools_testutils import is_live, recorded_by_proxy
 from utils import is_token_expiration_within_allowed_deviation
-from azure.identity import DefaultAzureCredential
 from acs_identity_test_case import ACSIdentityTestCase
-
+from azure.communication.identity import CommunicationIdentityClient
+from devtools_testutils.fake_credentials import FakeTokenCredential
+from azure.identity import DefaultAzureCredential
+from _shared.utils import get_http_logging_policy
 
 class ArgumentPasser:
     def __call__(self, fn):
         def _wrapper(test_class, _, value, **kwargs):
             fn(test_class, _, value, **kwargs)
+
         return _wrapper
 
 class TestClient(ACSIdentityTestCase):
     def setup_method(self):
         super().setUp()
 
-    @recorded_by_proxy
-    def test_create_user_from_managed_identity(self):
```

Comment:
the only remark I have is regarding the `managed identity` naming. it's not correct it should be sth like `test_create_user_from_token_credential` because we support all types of security principals (including managed identity and service principal) and all related token credentials...


I think in this case, the test uses [DefaultAzureCredential](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet) -> [EnvironmentCredential](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet) -> [ClientSecretCredential](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.clientsecretcredential?view=azure-dotnet) 

It's not super important, just a nice to have...we can also track it in a separate item...or just dismiss it entirely :)

Reply:
I'll address in my next PR that I'm currently working on